src/contributing.md

```diff
@@ -434,6 +434,15 @@ Just a few things to keep in mind:
   it might benefit from having a Table of Contents at the beginning,
   which you can auto-generate by including the `<!-- toc -->` marker at the top.
 
+#### ⚠️ Note: Where to contribute documentation changes
+
+If your change **only involves the documentation content of rustc-dev-guide** and **does not involve the rustc code itself**,
+please submit your changes and PRs directly to the [rust-lang/rustc-dev-guide](https://github.com/rust-lang/rustc-dev-guide) repository.
+Only changes with code changes in rust-lang/rust should be submitted to the [rust-lang/rust](https://github.com/rust-lang/rust) repository.
+
+This separation helps keep documentation and code maintenance independent, making both easier to manage and review.
+The complex merge process in `rust-lang/rust` can be avoided in this `rust-lang/rust-dev-guide` and can be immediately updated on [the documentation site](https://rustc-dev-guide.rust-lang.org/).
```

Comment:
Suggestion:

- Changes to `rustc-dev-guide` repo can be immediately reflected in the live `rustc-dev-guide`
- Changes to `rustc-dev-guide` repo do not need to go through bors CI in `rust-lang/rust`
- Less burden on bors queue in `rust-lang/rust`